packages/neos-ui/src/Sagas/UI/PageTree/index.js

```diff
@@ -78,26 +78,6 @@ export function * watchNodeCreated() {
     });
 }
 
-export function * watchReloadTree({globalRegistry, configuration}) {
-    const nodeTypesRegistry = globalRegistry.get('@neos-project/neos-ui-contentrepository');
-    yield takeLatest(actionTypes.UI.PageTree.RELOAD_TREE, function * reloadTree() {
-        const documentNodes = yield select(selectors.CR.Nodes.makeGetDocumentNodes(nodeTypesRegistry));
-        const {loadingDepth} = configuration.nodeTree;
-        const uncollapsedContextPaths = yield select(selectors.UI.PageTree.getUncollapsed, {loadingDepth});
-        const nodesToReload = documentNodes.toArray().filter(node =>
-            uncollapsedContextPaths.includes(node.get('contextPath')) &&
-            node.get('children').filter(child => nodeTypesRegistry.hasRole(child.get('nodeType'), 'document')).size
-        );
-
-        for (let i = 0; i < nodesToReload.length; i++) {
-            const node = nodesToReload[i];
-            const contextPath = node.get('contextPath');
-
-            yield put(actions.UI.PageTree.requestChildren(contextPath, {unCollapse: false}));
```

Comment:
OK. in the old code, we had many AJAX requests, now we only have one.

Reply:
exactly, works a lot faster